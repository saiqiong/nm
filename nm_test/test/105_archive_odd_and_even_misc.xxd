213c 6172 6368 3e0a 3120 2020 2020 2020 
2020 2020 2020 2020 3135 3230 3433 3138
3831 2020 3130 3935 3520 3020 2020 2020 
3130 3036 3434 2020 3720 2020 2020 2020 
2020 600a 636f 7563 6f75 0a0a 3220 2020 
2020 2020 2020 2020 2020 2020 3135 3230
3433 3138 3836 2020 3130 3935 3520 3020 
2020 2020 3130 3036 3434 2020 3620 2020 
2020 2020 2020 600a 7361 6c75 740a     
