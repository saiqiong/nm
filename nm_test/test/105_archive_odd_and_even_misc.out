!<arch>
1               1520431881  10955 0     100644  7         `
coucou

2               1520431886  10955 0     100644  6         `
salut
